### PR TITLE
Added AuthSendEmailSuccess.vue #380

### PR DESCRIPTION
**Pull request by giteshk11**
# Issue Being Addressed

#380 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For New Features:** What feature are we adding? Explain your technical approach to doing this.
Added new screen when ther user requests for forget password

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Click on forget password
2. Enter Email and click on the button
4. The new screen should be visible

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

### CODE REVIEW

**Comment by paulchrisluke** [comment]
this was successful, but when I press the link in the email I get 
https://deploy-preview-385--browtricks.netlify.app/reset?email=chris%40whynot.earth&token=CfDJ8Lmp%2FloC2NtHgpfrZlrtgOtvnbOb0URl7xp%2F0m3oJB3KUrD8NjuGloxqXdinTctKN4veCD%2Fs0gTasgmSQFWMKghF7e6hIu8wh1zHOws3oDlnWz%2Fzf9HGEFpfd8Ty1Ss1aAIho1SfdyFrlC4ubdAOKQbMCsE%2FLAzOQQkvY9pheI%2Bua%2FnmBuL3LjBV3u3BXLmVvg%3D%3D
image

**Comment by mort3za** [comment]
@giteshk11 

change `if (!this.userName || !this.token) {` to check also email in url. (for fixing the issue that chris mentioned)

**Review by paulchrisluke — APPROVED** [review]
lets merge this and test with more recent staging changes

File: src/views/AuthForgotPassword.vue [hunk 1]
```diff
@@ -1,40 +1,37 @@
...
-              autocomplete: 'username',
+              autocomplete: 'email',
               enterkeyhint: 'send',
-              name: 'username'
+              name: 'email'
             }"
-            :validatorModel="$v.userName"
-            :serverErrors="serverErrors.UserName"
+            :validatorModel="$v.email"
+            :serverErrors="serverErrors.email"
```
Comment (mort3za) [inline src/views/AuthForgotPassword.vue:28]: `serverErrors.Email` is correct

File: src/views/AuthResetPassword.vue [hunk 2]
```diff
@@ -102,7 +102,7 @@ export default {
     }
   },
   created() {
-    if (!this.userName || !this.token) {
+    if (!this.userName || !this.token || !this.email) {
```
Comment (mort3za) [inline src/views/AuthResetPassword.vue:105]: it should be `!(this.userName || this.email) || !this.token`.

File: src/views/AuthResetPassword.vue [hunk 3]
```diff
@@ -112,6 +112,9 @@ export default {
     },
     token() {
       return this.$route.query.token;
+    },
+    email() {
+      return this.$route.email;
```
Comment (mort3za) [inline src/views/AuthResetPassword.vue:117]: I don't see any usage of this computed variable. anyway it should be `return this.$route.query.email` is correct, we want to read the email query parameter in url.

**Review by mort3za — CHANGES_REQUESTED** [review]

**Comment by mort3za** [comment]
Please merge staging to your branch and test it again. I encountered this error: 

Sorry, This page is not valid. Please check the link and try again.

**Review by mort3za — APPROVED** [review]

**Pull request merged**